pkg/api/interface.go

```diff
@@ -44,17 +44,52 @@ package api
 
 const ServerPatchAnnotationName = "dual-pod.llm-d.ai/server-patch"
 
+// RequesterContainerName is the 'main' container in the server-requesting pod
+// whose job includes reporting the associated accelerators, exposing the
+// readiness from the server-running pod to server-requesting pod, etc.
+const RequesterContainerName = "requester"
+
+// InferenceServerContainerName is the name of the container which is described by the server patch.
+// This container is expected to run the inference server using vLLM.
+const InferenceServerContainerName = "inference-server"
+
+// ServerRunningPodNameSuffix is the suffix that is appended to the name of the
+// server-requesting Pod to form the name of the server-running Pod.
+const ServerRunningPodNameSuffix = "-server"
+
 // AdminPortAnnotationName is the name of an annotation whose value
-// is the name of the port on the "inference-server" container to be
+// is the name of the port on the server-requesting pod to be
 // queried to get the set of associated accelerators.
 const AdminPortAnnotationName = "dual-pod.llm-d.ai/admin-port"
 
+// AdminPortDefaultValue is the default port number of the server-requesting pod
+// to be queried to get the set of associated accelerators.
+const AdminPortDefaultValue = "8081"
+
+// PodRoleAnnotationName is the name of an annotation that
+// indicates the role of a Pod. The value of the annotation is
+// either "requester" for a server-requesting Pod,
+// or "runner" for a server-running Pod.
+const PodRoleAnnotationName = "dual-pod.llm-d.ai/role"
+
+// PodRoleAnnotationValueRequesting signals a server-requesting pod among the dual pods.
+const PodRoleAnnotationValueRequesting = "requester"
+
+// PodRoleAnnotationValueRunning signals a server-running pod among the dual pods.
+const PodRoleAnnotationValueRunning = "runner"
+
 // RunnerData is the data made available to the server patch.
 type RunnerData struct {
-  // NodeName is the name of the Node to which the Pod is bound
-  NodeName string
+	// NodeName is the name of the Node to which the Pod is bound
+	NodeName string
+
+	// GPUIndices is a string that represents accelerators by their indices.
+	GPUIndices string
```

Comment:
Why would a patch template need this variable?

The dual-pod controller understand the technique that it is using and the role that the CUDA_VISIBLE_DEVICES envar plays in that technique. And the server-requesting Pod author does not know the technique being used.

Reply:
Removed this from `RunnerData`.